packages/notebook/src/notebooklspadapter.ts

```diff
@@ -198,7 +198,6 @@ export class NotebookAdapter extends WidgetLSPAdapter<NotebookPanel> {
     if (this.isDisposed) {
       return;
     }
-    this._isDisposed = true;
```

Comment:
@fcollonval  The memory leak issue of the notebook adapter is fixed here, this line prevents the `super.dispose()` from being called correctly.

Reply:
Thanks :wink: